semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/rpc/RpcSemanticConventionUtilsTest.java

```diff
@@ -165,6 +166,23 @@ public void testGetGrpcXForwardedFor() {
     assertEquals("198.12.34.1", RpcSemanticConventionUtils.getGrpcXForwardedFor(event).get());
   }
 
+  @Test
+  public void testGetGrpcRequestMetadataHost() {
+    Event event = mock(Event.class);
```

Comment:
Also add one more test, if rpc system is not set to `Grpc`, the method returns to `Optional.Empty()`

Reply:
resolved